admin-manual/installation/installation.rst

```diff
@@ -446,7 +459,7 @@ Do not enable Elasticsearch if you are running Archivematica in headless mode.
    set -a -e -x
    source /etc/sysconfig/archivematica-dashboard
    cd /usr/share/archivematica/dashboard
-   /usr/lib/python2.7/archivematica/dashboard/bin/python manage.py syncdb --noinput
+   /usr/share/python/archivematica-dashboard/bin/python manage.py syncdb --noinput
```

Comment:
I don't think we use syncdb anymore right? We just need to call `migrate`. There are multiple calls to `syncdb` in this document, I think they all need to be changed.